core/primitives/src/types.rs

```diff
@@ -1,12 +1,14 @@
-use hash::{CryptoHash, hash, hash_struct};
-use signature::Signature;
+use hash::{hash, hash_struct, CryptoHash};
+use signature::{PublicKey, Signature};
 use std::borrow::Borrow;
 use std::hash::{Hash, Hasher};
 
 /// User identifier. Currently derived tfrom the user's public key.
 pub type UID = u64;
 /// Account alias. Can be an easily identifiable string, when hashed creates the AccountId.
 pub type AccountAlias = String;
+/// Public key alias. Used to human readable public key.
+pub type PublicKeyAlias = String;
```

Comment:
this is just a key id right? i think it may be a good idea to distinguish. otherwise, we need to clarify how we go from one to another

Reply:
It's a base58 of public key. What do you mean by key id?